### DISCUSSION TITLE
Calculated Metrics In Custodian

### DISCUSSION THREAD

**Discussion by sumodirjo** [cloud-custodian/cloud-custodian#7361] [General]
I'm trying to get an inventory of AWS resources based on the calculated metric. What I mean by calculated metrics here is new metrics that derived from 2 (or more) metrics that already available in CloudWatch.

For example, I want to get AWS Elasticache Memcached clusters that have Hit Rate of less than 80%.

Since HitRate metrics are not available in CloudWatch I have to calculate it as follow

HitRate = GetHits / (GetHits + GetMisses) * 100%

My question is: can I calculate metrics using cloud custodian? 

If this is not supported yet, is there any plan to support this capability?

## References

* https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/CacheMetrics.Memcached.html

**Comment by kapilt** [comment]
we support some basic calculations on the basis of a metric comparison to a resource attribute, ie to support things like percentage utilization (percent-attr, attr-multiplier options on metrics filter). we don't support comparisons between n metrics in arbitrary combinations.

wrt to plans to support it, like most features, it's on the basis of someone interested in contributing it.
